tests/aws/test_serverless.py

```diff
@@ -97,11 +121,13 @@ def test_dynamodb_stream_handler_deployed(self, aws_client, setup_and_teardown):
         assert event_source_arn == resp["Table"]["LatestStreamArn"]
 
     @markers.skip_offline
-    @markers.aws.unknown
+    @markers.aws.validated
     @pytest.mark.skip(reason="flaky")
```

Comment:
Q: Is this test still flaky?

Reply:
Don't know for sure - I only ran it a few times locally. But as there hasn't been any changes in kinesis lately, and the test was set as flaky in June I assume it's still the case.